### ISSUE TITLE
[BUG] Unable to move focus in the app list using keybpard

### ISSUE THREAD

**Issue by ruslaniv** [alienator88/Pearcleaner#372]
### Describe the bug:
Unable to move focus between the apps in the app list pane using keyboard navigation keys

### Steps to reproduce:
1. Open the app
2. Hit any arrow key on the keyboard
3. The focus does not change to the next / previous app
   

### Expected behavior:
The focus should move to the next / previous app as if I clicked on it

### Info:
 - OS: 26.0.1 (25A362)
 - Pearcleaner Version: 5.2.3

### Debug Console
[Oct 12, 6:15:38 PM] Updater: performing update check

**Comment by alienator88** [comment]
The list view uses a lazyVStack to show the apps and not a regular listview.
For that just tabbing through the items is supported out of the box. I would need to write a custom implementation to support arrow keys and I'm not super familiar with the accessibility portion regarding this functionality.

**Comment by ruslaniv** [comment]
But the way tabbing works now is useless:
1. It does not change the focus, if I click on item A and then tab to item B the focus still stays on A
2. If I tabbed to item D and then clicked on item B and then resume tabbing it will resume tabbing from item D (which is still useless since the focus never changes) but is super confusing
3. When I tab between apps and reach the last item in that row, the tab highlight will leave the app list pane and jump into the file list pane! And it will start highlighting the file items as i tab! I am not even sure why this is happening. 
4. I do not understand how tabbing mechanism chooses where to move the highlight next. Sometimes it will arbitrary reset it and start tabbing from the very beginning, sometimes it will continue tabbing form the previous item.

**Comment by alienator88** [comment]
Just fyi, I didn't add any of this tabbing mechanism, it's just how macos is handling it inside the app automatically.
Also from my understanding per Apple's documentation, Tab is only used to change the focus `rectangle`, not actually trigger an action/view when you Tab to it. When you are focused on an item, you can hit Spacebar to actually run the action though.

That behavior with the last item is because like I said I use a LazyVStack. The other items in the list are not actually there yet until you scroll, they are lazily loaded. So it makes sense why it's behaving that way, although not super helpful for your use-case.

I don't have the answers you're looking for here either, I don't have much experience with the accessibility side on macOS.

**Comment by ruslaniv** [comment]
OK, this is really not a big deal, especially if it is this hard to implement. 
It would be a little bit more convenient to use keyboard navigation but definitely not a deal breaker - I will happily continue using the app.
